docs/content/operations/including-extensions.md

```diff
@@ -3,31 +3,47 @@ layout: doc_page
 ---
 # Including Extensions
 
-Druid uses a module system that allows for the addition of extensions at runtime.
+Druid uses a module system that allows for the addition of extensions at runtime. To instruct Druid to load extensions, follow the steps below.
 
-## Specifying extensions
+## Download extensions
 
-Druid extensions can be specified in the `common.runtime.properties`. There are two ways of adding druid extensions currently.
+Core Druid extensions are already bundled in the Druid release tarball. You can get them by downloading the tarball at [druid.io](http://druid.io/).
+Unpack the tarball, you will see an ```extensions``` folder that contains all the core extensions, along with a ```hadoop-dependencies``` folder
```

Comment:
It's better not to use a comma to run two sentences together.  Here you could use a semicolon instead of a comma after "Unpack the tarball".  (The comma doesn't interfere with the meaning except by being jarring to some readers.)